build.gradle

```diff
@@ -29,6 +29,7 @@ minecraft {
 
 // Replace version in `mcmod.info`
 processResources {
+    duplicatesStrategy = 'include'
```

Comment:
Please add a space between the `duplicateStrategy` line and the `from()` line

Reply:
got it clean